src/test/meson.build

```diff
@@ -1042,6 +1042,11 @@ tests += [
          [],
          []],
 
+        [['src/libsystemd/sd-lldp/test-lldp.c'],
+         [libbasic,libshared_static,libsystemd_static
+          ],
```

Comment:
Please folllow the usual style: one per line, no concatenation without whitespace.